src/components/tabs/tabs.ts

```diff
@@ -265,7 +265,7 @@ export class Tabs extends Ion implements AfterViewInit, RootNode, ITabs {
 
   ngOnDestroy() {
     this._resizeObs && this._resizeObs.unsubscribe();
-    this.parent.unregisterChildNav(this);
+    this.parent && this.parent.unregisterChildNav && this.parent.unregisterChildNav(this);
```

Comment:
ok, let's trust TS a little bit, this.parent is already NavControllerBase, if `unregisterChildNav` is undefined we want this to crash (because it would be a bug) we don't want to silently omit it